src/components/LeadSection/LeadSectionCards/LeadSectionCards.tsx

```diff
@@ -0,0 +1,33 @@
+import cl from 'classnames';
+import { FC } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { leadSectionMockData } from '../../../_mockData/LeadSectionMockData';
+import s from './LeadSectionCards.module.scss';
+
+interface ILeadSectionCardsProps {}
+
+export const LeadSectionCards: FC<ILeadSectionCardsProps> = () => (
+  <ul className={cl(s.cards)}>
+    <li className={cl(s.card)}>
```

Comment:
есть ли какой-то резон выносить первую лишку отдельно здесь? т.е. нельзя ли написать что-то типа, убрав текст в mockData
```suggestion
    {leadSectionMockData.cardsData.map((card, index) => (
      <li className={cl(s.card)} key={uuidv4()}>
        <div className={cl(s.cardHeader)}>
          <p className={cl(s.cardTitleBig)}>{card.titleBig}</p>
          {card.titleSmall && <p className={cl(s.cardTitleSmall)}>{card.titleSmall}</p>}
        </div>
        <p className={cl(s.cardText)}>
          {card.text}{' '}
          {index === 0 && (
            <a href={leadSectionMockData.url} className={s.link}>
              ссылке
            </a>
          )}
        </p>
      </li>
    ))}
```

Reply:
обсуждалась что она статична и всегда должна быть первой. А с бэкенда ожидается только url. Поэтому проще всего было сделать её отдельно до map.